TCAT/Cells/RouteTableViewCell.swift

```diff
@@ -264,10 +264,14 @@ class RouteTableViewCell: UITableViewCell {
                 
                 let delayedDepartTime = departTime.addingTimeInterval(TimeInterval(delay))
                 
-                if Time.compare(date1: departTime, date2: delayedDepartTime) == .orderedAscending {
-                    return .late(date: delayedDepartTime)
+                if Time.compare(date1: departTime, date2: delayedDepartTime) == .orderedAscending { // bus is delayed
+                    if (delayedDepartTime >= Date() || delay >= 120) {
+                        return .late(date: delayedDepartTime)
+                    } else { // delay < 120
+                        return .onTime(date: departTime) // what time??
```

Comment:
what does the `// what time?` comment mean

Reply:
whoops it was for when I was asking you what time to display in that case, I just removed it